modules/bot/package.json

```diff
@@ -5,7 +5,7 @@
   "main": "dist/src/index.js",
   "scripts": {
     "build": "rm -rf dist && tsc --project tsconfig.json",
-    "start": "node dist/src/index.js"
+    "start": "node --inspect=0.0.0.0:9229 dist/src/index.js"
```

Comment:
It's useful to be able to profile bots directly, so this directive allows us hook them up to a profiler.